packages/table/src/headers/_headers.scss

```diff
@@ -254,3 +252,32 @@
   }
   // stylelint-enable selector-class-pattern
 }
+
+.bp-table-column-name,
+.bp-table-row-name {
+  transition: color $cell-transition-duration;
+}
+
+.bp-table-header.pt-loading {
+  @include cell-content-align-vertical();
+
+  .bp-table-column-name,
+  .bp-table-row-name {
+    flex: 1;
+  }
+
+  .bp-table-column-name-text {
+    @include cell-content-align-vertical();
+    padding: floor(($column-header-min-height - $column-name-text-skeleton-height) / 2)
+      $pt-grid-size
+      ceil(($column-header-min-height - $column-name-text-skeleton-height) / 2);
```

Comment:
this entire property is actually worthless because of the mixin. try removing it and notice how the skeleton becomes perfectly centered (and how it was clearly a pixel off with this padding).

i'd still suggest 8px height cuz even numbers are always better for centering.